tests/cli/test_create.py

```diff
@@ -2,20 +2,18 @@
 
 import pexpect
 
-from ethpm_cli.constants import ETHPM_CLI_VERSION, SOLC_INPUT
+from ethpm_cli.constants import SOLC_INPUT
 
 
 def test_custom_manifest_builder(tmp_project_dir, test_assets_dir):
     child = pexpect.spawn(
-        f"ethpm create manifest-wizard --project-dir {tmp_project_dir}", timeout=5
+        f"ethpm create wizard --project-dir {tmp_project_dir}", timeout=5
     )
-    child.expect(f"ethPM CLI v{ETHPM_CLI_VERSION}\r\n")
+    child.expect(f"A command line tool for the Ethereum Package Manager.")
```

Comment:
Maybe make this a constant you can import so the text isn't reproduced across all these files.